test/logger_test.rb

```diff
@@ -455,9 +455,9 @@ class TestLogger < Hanami::Logger; end
           output =
             stub_stdout_constant do
               class TestLogger < Hanami::Logger; end
-              TestLogger.new.info('This is a message with spaces')
+              TestLogger.new.info("This is a message with quotes \" ' and with spaces")
```

Comment:
@jodosha I've updated the PR, I use gsub to replace '"' with '\"', but the only problem I see I have to check the message with `is_a?`, but it works now.